### ISSUE TITLE
Apply w-full class to Float div blank rendered

### ISSUE THREAD

**Issue by IsidroMar95** [ycs77/headlessui-float#18]
**Use Version**
 - Headless UI: v1.6.0
 - Headless UI Float: v0.7.0
 - Framework: vue v3.2.29

**Describe the question**
I want my ListboxOptions take full width of the parent element, but I cant.

```js
        <Float
            as="div"
            data-testid="float"
            class="w-full"
            placement="bottom"
            :offset="2"
            :shift="2"
            :flip="10"
            :z-index="10"
            tailwindcss-origin-class
        >
          <ListboxButton
              class="relative w-full bg-white">
          </ListboxButton>
          <ListboxOptions
              class="z-10 mt-1 w-full">
            <ListboxOption
                as="template"
                v-slot="{ active, selected }"
            >
              <li class="w-full">
              </li>
            </ListboxOption>
          </ListboxOptions>
        </Float>
```

![image](https://user-images.githubusercontent.com/23019968/187296021-79203d70-3e96-42bf-a49a-e6fb3b948732.png)

**Comment by tung-eh** [comment]
Yeah, the library should provide users the ability to inject style to the floating wrapper element IMO.

**Comment by ycs77** [comment]
Hi! Now released v0.9.0 & documentation, can see the [Adaptive width](https://headlessui-float.vercel.app/vue/adaptive-width.html) section to using.
